### PR TITLE
Add known issues to changelogs

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/install/desktop-app-changelog.rst [hunk 1]
```diff
@@ -106,6 +106,7 @@ All Platforms
 Known Issues
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+- Mattermost Desktop App v5.1.0 cannot be launched on a Windows multi-user system.
```
Comment (cwarnermm) [inline source/install/desktop-app-changelog.rst:109]: I'm not 100% clear on what a "Windows multi-user system" means. Does it mean a Windows system that contains more than a single user account?

**Review by cwarnermm — COMMENTED** [review]
Thanks, @amyblais. I've provided feedback in the form of a question inline.

File: source/install/desktop-app-changelog.rst (hunk 1 above)
Comment (amyblais) [reply source/install/desktop-app-changelog.rst:109]: Likely Devin can help with details (related to https://github.com/mattermost/desktop/issues/2117).

Comment (cwarnermm) [reply source/install/desktop-app-changelog.rst]: Thanks, @JtheBAB!

**Pull request merged**